easybuild/easyblocks/c/cuda.py

```diff
@@ -75,7 +75,7 @@ def install_step(self):
         if 'DISPLAY' in os.environ:
             os.environ.pop('DISPLAY')
 
-        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True)
+        run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True, maxhits=1500)
```

Comment:
Sure, just add a comment maybe why you're overriding the default value, and what is means (300s wait for nothing to change in the output without seeing a known question)